frontend/src/views/consumer/filter-view/index.js

```diff
@@ -0,0 +1,193 @@
+      tween({ ...props, duration: 700, ease: easing.easeOut }),
+  },
+});
+
+const Wrapper = styled(Sliddable)`
+  position: fixed;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  top: 0;
+  left: 0;
+  height: 100%;
+
+  z-index: 10;
+  background-image: linear-gradient(to top, #e6e9f0 0%, #eef1f5 100%);
+`;
+const ScrollContainer = styled.div`
+  overflow-y: auto;
+  overflow-x: hidden;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  padding: 2rem 1rem;
+  padding-top: 7rem;
+
+  & > div {
+    margin: 0.5rem;
+    flex-shrink: 0;
+  }
+  & > button {
+    align-self: flex-end;
+    margin: 1rem;
+    flex-shrink: 0;
+  }
+`;
+
+export default withTheme(
+  connect('store')(
+    class FilterView extends Component {
+      setAgeGroupFilter = value => {
+        this.props.store.filters.setFilters('ageGroupLimit', value);
+      };
+      setAreaFilter = value => {
+        this.props.store.filters.setFilters('area', value);
+      };
+      setDateFilter = value => {
+        this.props.store.filters.setFilters('date', value);
+      };
+      setEventTypeFilter = value => {
+        this.props.store.filters.setFilters('eventType', value);
+      };
+
+      render() {
+        const { theme, store } = this.props;
+        const area = [
+          'Tikkurila',
+          'Aviapolis',
+          'Myyrmäki',
+          'Korso',
+          'Hakunila',
+          'Koivukylä',
+        ];
+
+        return (
+          <Wrapper pose={store.ui.filterViewActive ? 'enter' : 'exit'}>
+            <ScrollContainer>
+              <Typography type="title" color={theme.palette.primaryDeep}>
+                Rajaa hakua
+              </Typography>
+              <div>
+                <Typography type="subheader">Lapsen ikä</Typography>
+                <TagPillGroup
+                  value={store.filters.ageGroupLimit}
+                  onChange={this.setAgeGroupFilter}
+                  highlightColor={theme.palette.primaryDeep}
+                  tags={[
+                    { value: '0-3', label: '0-3' },
+                    { value: '3-6', label: '3-6' },
+                    { value: '6-12', label: '6-12' },
+                    { value: '13+', label: '13+' },
+                  ]}
+                />
+              </div>
+              <div>
+                <Typography type="subheader">Alue</Typography>
+                <TagPillGroup
+                  value={store.filters.area}
+                  onChange={this.setAreaFilter}
+                  highlightColor={theme.palette.primaryDeep}
+                  tags={area.map(a => ({ value: a, label: a }))}
+                />
+              </div>
+              <div>
+                <Typography type="subheader">Päivämäärä</Typography>
+                <TagPillGroup
+                  highlightColor={theme.palette.primaryDeep}
+                  value={store.filters.date}
+                  onChange={this.setDateFilter}
+                  tags={[
```

Comment:
I think this list would be better to be defined elsewhere as a constant.

Reply:
then im going to pull this out of the component